packages/neon-core/src/wallet/Account.ts

```diff
@@ -88,6 +88,20 @@ export class Account implements NeonObject<AccountJSON> {
     });
   }
 
+  public static validateKey(str: string): string | void {

```

Comment:
Instead of string | void, consider either:

- narrowing the return type to the specific strings (eg. 'PrivateKey' | 'EncodedPublicKey' | ...)
- Adding an enum to consts and referencing that